app/Website/src/Service/MailboxLayerService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Website\Service;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Support\Facades\Config;
+
+class MailboxLayerService
+{
+    protected Client $client;
+    protected string $baseUrl = 'http://apilayer.net/api';
```

Comment:
You can put this in an environment variable and load it from the custom config file.